app/grandchallenge/workstation_configs/templatetags/get_help_text.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_help_text(obj, field):
+    return obj.get_help_text(field)
```

Comment:
Nitpick but it would be easier for re-use if this tag did `obj._meta.get_field(field).help_text` rather than having to define `get_help_text` on the model to use it.

Reply:
That would mean it accesses a 'protected' member of the object. Would that still be okay?